src/storage/query/QueryBaseProcessor.inl

```diff
@@ -335,12 +335,23 @@ void QueryBaseProcessor<REQ, RESP>::collectProps(RowReader* reader,
         }
         if (reader != nullptr) {
             const auto& name = prop.prop_.get_name();
+            VariantType v;
             auto res = RowReader::getPropByName(reader, name);
             if (!ok(res)) {
-                VLOG(1) << "Skip the bad value for prop " << name;
-                continue;
+                VLOG(1) << "Bad value for prop " << name;
+                // TODO: Should return NULL
+                auto defaultVal = RowReader::getDefaultProp(prop.type_.type);
+                if (!defaultVal.ok()) {
+                    // Should never reach here.
+                    LOG(FATAL) << "Get default value failed for " << name;
```

Comment:
if someone adds new value type and doesn't modify the `getDefaultProp`, here will crash.

Reply:
We want it crash exactly. Someone add new type, must add the corresponding test.

Reply:
It might  be discovered by user